client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/contact-form/style.scss

```diff
@@ -35,9 +36,13 @@
 
 .signup-multi-step-form__name-fields {
     display: flex;
-    flex-direction: row;
+    flex-direction: column;
 	gap: 16px;
 
+	@include break-medium {
```

Comment:
Let's use break-small to target devices <600px, this targets upto 780px which is a tablet device.